lib/matplotlib/legend_handler.py

```diff
@@ -110,10 +112,10 @@ def legend_artist(self, legend, orig_handle,
 
         """
         xdescent, ydescent, width, height = self.adjust_drawing_area(
-                 legend, orig_handle,
-                 handlebox.xdescent, handlebox.ydescent,
-                 handlebox.width, handlebox.height,
-                 fontsize)
+            legend, orig_handle,
```

Comment:
Please leave the double indent - this makes it easier to tell its an indent vs a code block...

Reply:
will be changed asap! :)